### DISCUSSION TITLE
How difficult is it to add an unsupported light to the integration?

### DISCUSSION THREAD

**Discussion by sabolcik** [cryptk/haomnilogic-local#125] [General]
The integration works great, but my lights show up as not supported.  How difficult is it to add unsupported lights and other than this generic error in the logs, what additional information would be helpful to share to identify the light type and next steps?  I don't get any lights that show up with either my pool or spa device that is created in the integration.  All other functionality works as expected.

This error originated from a custom integration.

Logger: custom_components.omnilogic_local.light
Source: custom_components/omnilogic_local/light.py:65
integration: OmniLogic Local (documentation, issues)
First occurred: 10:30:19 PM (2 occurrences)
Last logged: 10:30:19 PM

Your system has an unsupported light, this light may not function properly, please raise an issue: https://github.com/cryptk/haomnilogic-local/issues

**Comment by cryptk** [comment]
This is a long time later, but I just released 1.0.0 of the integration and it adds support for more lights including ColorLogic 4.0, Pentair lights and Zodiak lights.

CL 4.0 and Pentair have been tested and confirmed as working, but I don't know anyone with Zodiak lights. If that's what you have, feel free to open an issue and let me know if they are working with 1.0.0!
